gtsam_integrator/plugin.cpp

```diff
@@ -176,8 +176,11 @@ class gtsam_integrator : public plugin {
 
         assert(_pim_obj != nullptr && "_pim_obj should not be null");
 
+        // TODO last_imu_offset is 0, t_offset only take effects when it's negative.
```

Comment:
Does this make any difference in quality?

Reply:
For EuRoc, `t_offset` is very small (less than 1ms) so no difference made. For ZED, this value ranges from -10ms to -60ms and a big difference there could be. My experience is that not adding this offset gives a better headset experience. I wonder if ZED needs a calibration to update the camera extrinsics, intrinsics, gyroscope/accelerator noises and random walks, and so on. These values contribute to the estimates of the `t_offset` which is so off right now.